pkg/kademlia/kademlia.go

```diff
@@ -0,0 +1,36 @@
+package kademlia
+
+import (
+	"context"
+
+	"storj.io/storj/protos/overlay"
+)
+
+type nodeID string
+
+// DHT is the interface for the DHT in the Storj network
+type DHT interface {
+	GetNodes(ctx context.Context) []overlay.Node
```

Comment:
this returns, what, all nodes? all 100k? everything in the routing table? the routing table might be big

Reply:
Pagination makes the most sense to me.